values.yaml

```diff
@@ -50,25 +64,21 @@ zabbixserver:
     # -- Zabbix server Docker image name
     repository: zabbix/zabbix-server-pgsql
     # -- Tag of Docker image of Zabbix server
```

Comment:
Hi @sa-ChristianAnton!

With tag centralized, this line can be removed, correct?